app/pdf-viewer/buffer.py

```diff
@@ -203,7 +203,7 @@ def set_focus_text(self, new_text):
 
     def get_toc(self):
         result = ""
-        toc = self.buffer_widget.document.getToC()
```

Comment:
fitz.Document, this library use getToC() to get table, did you know this is library API?